integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -32,4 +35,17 @@ public void webhookSiteContainsRequest(String requestText, int timeoutMinutes) {
                                 requestTextWithoutPlaceholders)
                         .anyMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders)));
     }
+
+    @Then("^Webhook site does not contains request with text \"([^\"]*)\" within (\\d+) seconds$")
+    public void webhookSiteDoesNotContainsRequest(String requestText, long timeoutSeconds) throws InterruptedException {
+        String requestTextWithoutPlaceholders = ContextResolver.resolveWithScenarioContext(context, requestText);
+        BooleanSupplier failureCondition =
+                () -> WebhookSiteResource.requests().stream().map(WebhookSiteRequest::getContent).noneMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders));
+        Duration duration = Duration.ofSeconds(timeoutSeconds);
+        Instant timeoutTime = Instant.now().plus(duration);
+        while (timeoutTime.isAfter(Instant.now())) {
+            Thread.sleep(duration.toMillis());
```

Comment:
This line is marked as code smell by sonar. I guess wee need to use wait mechanism as @sutaakar did here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/652